doc/nrf/ug_nrf5340.rst

```diff
@@ -567,6 +567,27 @@ See the following instructions.
    :start-after: fota_upgrades_start
    :end-before: fota_upgrades_end
 
+Simultaneous multi-image DFU
+****************************
+
+The simultaneous update of multiple images is available for testing since |NCS| v1.7.0.
+It allows both the application core and the network core to be updated at the same run.
+
+To enable the simultaneous update of multiple images set the following options:
+
+* :kconfig:`CONFIG_BOOT_UPGRADE_ONLY` - The simultaneous update of multiple images does not support network core image reversion so it is recommended to disable application image reversion.
```

Comment:
I think these options would not be enough (configuration is a bit more complicated). I would refer to existing samples for Thingy:53 and test that was introduced by Hakon instead of going into defails here.

Reply:
e.g. we must enable FLASH simulator for MCUboot and configure it properly.
The external FLASH must be used too I think

Reply:
The reference to thingy is at line 583.

Which other options are needed?

Reply:
Ignore anything related to flash configuration etc. Only the bootloader options. Only specific for this feature (so no multithreading or anything like that).